src/telemetry/filters.ts

```diff
@@ -1,5 +1,4 @@
 import { ITelemetryItem } from '@microsoft/applicationinsights-web';
-import { exception } from 'console';
```

Comment:
Why was this removed? It doesnt seem to be related to the task or anything in the PR

Reply:
It was causing a linting error so I removed it

Reply:
Sounds good!